client/static/persist.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta charset="UTF-8">
+</head>
+
+<body>
+
+<script>
+
+// Note: This script is not compiled, so make sure everything here is able to
```

Comment:
Why is it not compiled?

Reply:
Probably should be, it's just so small that not compiling it seems like an easier alternative to configuring another entry for Webpack. Should probably run it at least through closure compiler for minification though. Also should research caching performance of embedded script vs a separate one.

Reply:
You can easily embed compiled scripts.

Reply:
Yea I know with the html plugin, my main concern is that the Webpack polyfills are bigger than necessary.